src/pg/test/e2e.test.js

```diff
@@ -439,6 +439,81 @@ describe('pg_e2e', () => {
     expect(res2).toEqual(null);
   });
 
+  describe('val', () => {
+    const id = uuid();
+    // beforeAll(async () => {
+    //   const result = await store.write('users', [
+    //     {
+    //       $key: id,
+    //       $put: true,
+    //       name: 'A',
+    //       email: 'a',
+    //       settings: {
+    //         foo: {
+    //           bar: 'incorrect',
+    //           baz: 'correct'
+    //         }
+    //       }
+    //     },
+    //   ]);
+    // });
+    // afterAll(async () => {
+    //   await store.write(['users', id], null)
+    // })
+    test('key is removed', async () => {
+      await store.write('users', [
+        {
+          $key: id,
+          $put: true,
+          name: 'A',
+          email: 'a',
+          settings: {
+            foo: {
+              bar: 'incorrect',
+              baz: 'correct'
+            }
+          }
+        },
+      ]);
+      const res1 = await store.write(['users', id], {
+        settings: {
+          foo: {
+            bar: null,
+          }
+        }
+      });
+      expect(Object.keys(res1.settings.foo)).toEqual(['baz']);
+      await store.write(['users', id], null)
```

Comment:
If i don't reset this to null, 👇 test cases fails. Instead of `{name: 'A' }` we get `{name: null}`
https://github.com/usegraffy/graffy/blob/7e3f340cf08ffaea9a7a9ab7e23df3e9b9fa358e/src/pg/test/e2e.test.js#L727

Reply:
This stuff can be in beforeAll / afterAll